hex-number/hex-number.py

```diff
@@ -0,0 +1,102 @@
+class Node:
```

Comment:
Your pull request: HEXNUMBER-1021 Added hex-number.py and test-hex-number.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'ITERATOR', 'REQUESTS']

Your commit: HEXNUMBER-1021 Added hex-number.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'ITERATOR', 'REQUESTS']
Your commit: HEXNUMBER-1021 Added test-hex-number.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'ITERATOR', 'REQUESTS']